README.md

```diff
@@ -1,81 +1,180 @@
 # Make It So
 
-Make It So is a sample Todo app powered by Gemini AI, Firebase auth and firestore.
+Welcome to the _Make It So_ sample app, an AI-powered web app for to-do lists. It's an [Angular](https://angular.io/) app built using the Gemini API and Firebase.
 
-## Introduction
+The [Gemini API](https://ai.google.dev/gemini-api) gives you access to Google's latest
+generative AI models – the Gemini family of multimodal models. This _Make It So_ web app
+calls the Gemini API to generate a task list from either text or an image provided by
+the end user.
 
-[Read more about Firebase](https://firebase.google.com/docs)
-[Read more about Gemini](https://ai.google.dev/gemini-api/docs/quickstart?lang=node)
+The backend of _Make It So_ is powered by [Firebase](https://firebase.google.com/),
+which is Google's platform for building fullstack multi-platform apps. This app uses
+[Firebase Authentication](https://firebase.google.com/products/auth) for signing-in
```

Comment:
Should we mention the type of auth?

Reply:
I think in this opening paragraph, we should keep it short.

Later in the "Explore the app's codebase" section, lemmie add some additional details and a direct link to the anonymous auth docs.